src/test/java/com/gachigang/ontherun/controller/DepartmentControllerTest.java

```diff
@@ -20,12 +23,46 @@ public class DepartmentControllerTest {
     private DepartmentController departmentController;
 
    @Test
-    void getDepartmentByUserId() {
+    void testGetDepartmentByUserId() {
         User user = User.builder()
                         .id(1L)
                         .lastname("test")
                         .build();
         departmentController.getDepartmentByUserId(user);
         verify(departmentService, times(1)).getDepartmentByUserId(user.getId());
     }
+
+    @Test
+    void testGetDepartmentsByBusinessIdValidId() {
```

Comment:
Oh, sorry I guess @Konoplastiy and I have missed it :( In controller we have only one method call and 0 bussines logic, so it is not right to write tests on service method but store it in cotroller -- Hence, you did it right, and tests you have added need to be addresed to service (You can simply move them with small refactoring )

Reply:
I've remove it